full_bubble_model.py

```diff
@@ -103,6 +116,84 @@ def _colorTF(boolean):
 print(f'enable heat transfer: {_colorTF(enable_heat_transfer)}\tenable evaporation: {_colorTF(enable_evaporation)}\tenable reactions: {_colorTF(enable_reactions)}\tenable dissipated energy: {_colorTF(enable_dissipated_energy)}')
 if target_specie not in par.species:
     print(colored(f'Error, target specie \'{target_specie}\' not found in parameters.py', 'red'))
+
+
+"""________________________________Control parameters________________________________"""
+
+cpar_rules = dict(
```

Comment:
The existance, type and value of control parameters (in cpar) is now properly checked. A correct cpar is now enforced, no default values (e.g. when mu_L is missing). Proper error messages